hasher-matcher-actioner/scripts/submit_content_test.py

```diff
@@ -0,0 +1,165 @@
+#! /usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import os
+import json
+import base64
+import requests
+import dataclasses
+import typing as t
+from urllib.parse import urljoin
+from script_utils import HasherMatcherActionerAPI
+
+from hmalib.common.evaluator_models import Action, ActionRule
+from hmalib.common.classification_models import Label, ActionLabel, ClassificationLabel
+from hmalib.common.actioner_models import ActionPerformer, WebhookGetActionPerformer
+
+
+class DeployedInstanceTestHelper:
+    """
+    Class around testing a deployed instance of HMA from Content Submission to Hash - Match - Action
```

Comment:
I'm not able to understand the purpose of this class.
* The public methods are merely proxying their arguments to `self.api`. 
* What's the value that this class is adding? The constant for SAMPLE_PG_ID? Even that is used in a separate public function, not a method of this class.
* Given that `_refresh_token()` is also a private method of the `HasherMatcherActionerAPI`, it seems the staleness of the token is an internal matter of that class.

Reply:
This class is allowing self.api to stay at a low enough level it can be used independently of `hmalib`
This class is handles the specifics of `hmalib`

Reply:
However, for the 2nd two points I agree:
- removed the constant from the class 
- added docstring todo for eventual adjustment of refresh responsibility.

Reply:
> This class is allowing self.api to stay at a low enough level it can be used independently of `hmalib`
> This class is handles the specifics of `hmalib`

This module (submit_content_test) **imports** hmalib. So you can't use it without having hmalib on `PYTHONPATH`. If you want to avoid imports at the callsite, I'm not seeing what problem that's trying to solve.